packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -1,229 +1,412 @@
-.aa-ItemActionButton {
-  color: rgba(0, 0, 0, 0.4);
-}
-
-.aa-ItemActionButton {
-  background-color: transparent;
-  border: 0;
-  cursor: pointer;
-  opacity: 0.8;
-  padding: 0.5rem;
-  transition: color 100ms;
-  @media screen and (prefers-reduced-motion: reduce) {
-    transition: none;
-  }
-
-  &:hover,
-  &:focus {
-    color: rgba(80, 80, 80, 1);
-  }
+// ----------------
+// Darkmode
+// ----------------
+body {
+    &[data-theme='dark'],
+    &.dark {
+        --primary-color: rgb(247, 247, 0);
+        --muted-color: rgba(167, 167, 96, 0.6);
+        --selected-color: rgba(247, 247, 0, 0.1);
+        --icon-color: rgb(31, 207, 248);
+        --text-color: rgb(183, 192, 199);
+        --background-color: rgb(21, 23, 42);
+        --panel-shadow: inset 1px 1px 0 0 rgb(44, 46, 64), 0 3px 8px 0 rgb(0, 3, 9);
+    }
 }
 
-.aa-TouchOverlay {
-  backdrop-filter: blur(16px);
-  background: rgba(255, 255, 255, 0.95);
-  bottom: 0;
-  left: 0;
-  padding: 0.5rem;
-  position: fixed;
-  right: 0;
-  top: 0;
+// ----------------
+// Autocomplete
+// ----------------
+.aa-Autocomplete {
+    font-size: var(--font-size);
+    text-align: left;
+    line-height: 1em;
+    //reset
+    * {
+        box-sizing: border-box;
+        margin: 0;
+        padding: 0;
+    }
+    button {
+        appearance: none;
+        background: none;
+        border: none;
+    }
+    // searchbox
+    form.aa-Form {
+        position: relative;
+        display: flex;
+        border: 1px solid var(--muted-color);
+        align-items: center;
+        padding: 0 var(--spacing) 0 var(--spacing-half);
+        line-height: 1em;
+        background-color: var(--background-color);
+        border-radius: 3px;
+        &,
+        &.aa-Form--focused {
+            border-color: var(--primary-color);
+            box-shadow: var(--selected-color) 0 0 0 3px, inset var(--selected-color) 0 0 0 2px;
+            outline: currentcolor none medium;
+        }
+        .aa-InputWrapperPrefix {
+            order: 1;
+            flex-shrink: 0;
+            flex-wrap: none;
+            display: flex;
+            align-items: center;
+            padding-right: var(--spacing-half);
+            // the container for search and loading icons
+            label.aa-Label,
+            .aa-LoadingIndicator {
+                cursor: initial;
+                flex-shrink: 0;
+                text-align: center;
+                width: calc(var(--spacing) + var(--icon-size));
+                svg {
+                    color: var(--primary-color);
+                    stroke-width: var(--icon-stroke-width);
+                    width: 20px;
+                    position: relative;
+                    top: 1px;
+                }
+            }
+        }
+        .aa-InputWrapper {
+            position: relative;
+            width: 100%;
+            order: 3;
+            // input of the searchbox, where the placeholder and query appear
+            input.aa-Input {
```

Comment:
same with all of these element-specific styles, there's no aa-Input that's not an input